src/components/CourseCard/CompletedCourseCard.jsx

```diff
@@ -12,12 +12,14 @@ const CompletedCourseCard = (props) => {
       >
         View Course
       </a>
-      <a
-        className="btn btn-block btn-outline-primary btn-course-link"
-        href={props.linkToCourse}
-      >
-        View Certificate
-      </a>
+      {props.linkToCertificate && (
+        <a
+          className="btn btn-block btn-outline-primary btn-course-link"
+          href={props.linkToCourse}
```

Comment:
Did you intend to use `linkToCertificate`?

Reply:
Yup, good catch!